http/src/main/resources/init-resources/jupyter-docker-compose-gce.yaml

```diff
@@ -37,18 +37,6 @@ services:
       - /var/custom_env_vars.env
     # See https://docs.docker.com/engine/reference/run/#user-memory-constraints
     mem_limit: ${MEM_LIMIT}   # hard limit on memory consumption by the container
-  welder:
```

Comment:
hmm...I did start with duplicating `welder-docker-compose` and `proxy-docker-compose`, but figured it'd be nicer if we can just have one `-gce` and non gce compose file, and have separate network seting appropriately. Since AOU complicates things a bit, I was thinking we can just have a separate `crypto-detector-docker-compose.yaml` that we attach conditionally

Reply:
I thought it was better if each `*-docker-compose.yaml` file was independent and didn't reference each other. That way we can apply them conditionally without worrying about dependencies. We already have separate GCE and non-GCE `clusterResources` in `reference.conf` so there isn't much extra templating needed.